.gitignore

```diff
@@ -0,0 +1,2 @@
+small.dat
```

Comment:
small.dat is included on purpose, as a small example to run and test that everything is working.

Reply:
That was a mistake!  Sorry about that, I'll fix.

Reply:
Good catch, thanks.

Reply:
The reason I added it to `.gitignore` was so that when you run the code and it changes `small.dat` you don't accidentally commit the changes by doing `git add .`, which although not great practice, is definitely common practice.  Since `small.dat` is already tracked, the idea was this would just ignore any *further* changes.

However, I think my logic was wrong about how tracking works, so I don't think that adding this to the `.gitignore` does *anything*.  So I'll just remove it.